csharp/test/Apache.Arrow.Flight.Tests/FlightTests.cs

```diff
@@ -312,5 +312,26 @@ await foreach(var recordBatch in getStream.ResponseStream)
             ArrowReaderVerifier.CompareBatches(expectedBatch1, resultList[0]);
             ArrowReaderVerifier.CompareBatches(expectedBatch2, resultList[1]);
         }
+
+        [Fact]
+        public async Task EnsureTheSerializedBatchContainsTheProperTotalRecordsAndTotalBytesProperties()
+        {
+            var flightDescriptor1 = FlightDescriptor.CreatePathDescriptor("test1");
+            var expectedBatch = CreateTestBatch(0, 100);
+            var expectedTotalBytes = expectedBatch.Arrays.Sum(arr => arr.Data.Buffers.Sum(b => b.Length));
+
+            List<FlightInfo> expectedFlightInfo = new List<FlightInfo>();
+
+            expectedFlightInfo.Add(GivenStoreBatches(flightDescriptor1, new RecordBatchWithMetadata(expectedBatch)));
+
+            var listFlightStream = _flightClient.ListFlights();
+
+            var actualFlights = await listFlightStream.ResponseStream.ToListAsync();
+            var result = actualFlights.First();
+
+            //Expected values can be found in the FlightHolder class
```

Comment:
```suggestion
```
I'm not sure this comment is still needed since it appears you moved away from hard-coded values.